### DISCUSSION TITLE
Anyone know the flag for enabling the queue?

### DISCUSSION THREAD

**Discussion by Pizzo8** [PoomSmart/YTABConfig#61] [General]
it was available for premium users

**Comment by xzuyn** [comment]
also wondering about this.

**Comment by tonn333** [comment]
What does it do excatly?

**Reply by Pizzo8** [reply]
enable the queue feature like the one you get on desktop

**Comment by evkaw** [comment]
any updates on this? i want to at least be able to disable that annoying premium button

**Comment by D3XX3R** [comment]
Seems like this feature is now public but still behind a paywall. Is the flag known to anyone?

**Comment by GH05Tin** [comment]
Bump, this would be awesome.

**Comment by Invisible40** [comment]
Looking for the same

**Comment by correii** [comment]
Update on this:

The “add to queue” option is present in current builds of the app. You still can’t use it without premium though.

I really hope someone makes a tweak to bypass the paywall.

My guess is that the app is making a request to YT servers to check if the current user has premium, probably at startup and user switch. 

All it would take is altering that check to ensure it always returns yes, and all client side premium features would be enabled.

**Reply by D3XX3R** [reply]
It could be, that the video queue is not actually local on the device but rather on YouTube itself to be able to sync it with other devices.
That would make it impossible to bypass server-sided checks. Someone would have to implement an own queue system for it to work.

**Reply by elzii** [reply]
it's not. it's not even tab synced on desktop. it's locally stored and keyed with epoch stamps so doff tabs can have diff queues, and hard refreshes lose the Q
